### DISCUSSION TITLE
Adding Labels depending on Package Version

### DISCUSSION THREAD

**Discussion by adam-moss** [renovatebot/renovate#13178] [Request Help] [answered]
Apologies if this is obvious but I'm struggling to see it.

What would be the correct syntax to use `addLabels` inconjunction with the package version.

So, for example, if I wanted to add a label when a certain package was at a version < 2.16.0 but not if it were equal too or higher?

Thanks!

**Comment by rarkins** [comment, marked as answer]
Can you give an example of two different PRs where one would need the label and one not? Possibly matchCurrentVersion inside a package rule would suit this requirement

**Reply by adam-moss** [reply]
Sure.

If we take something like the current `log4shell` issue, then if say `log4j-core` is < v2.16.0 I'd want to apply the `log4shell` label, but if it is >= then (currently) I wouldn't.

Does that make sense?

Thanks!

**Reply by rarkins** [reply]
A package rule with matchCurrentVersion and labels should work
